py/desispec/qa/qa_quicklook.py

```diff
@@ -1405,41 +1462,31 @@ def run(self,*args,**kwargs):
         if "qafig" in kwargs: qafig=kwargs["qafig"]
         else: qafig = None
         
-        return self.run_qa(input_frame,camera,expid,paname=paname,skymodel=skymodel,amps=amps,dict_countbins=dict_countbins,url=url,qafig=qafig)
+        return self.run_qa(input_frame,paname=paname,skymodel=skymodel,amps=amps,dict_countbins=dict_countbins,url=url,qafig=qafig)
 
 
-    def run_qa(self,frame,camera,expid,paname=None,skymodel=None,amps=False,dict_countbins=None,url=None,qafig=None):
+    def run_qa(self,frame,paname=None,skymodel=None,amps=False,dict_countbins=None,url=None,qafig=None):
         from desispec.sky import qa_skysub
         from desispec import util
 
         if skymodel is None:
             raise IOError("Must have skymodel to find residual. It can't be None")
         #- return values
         retval={}
-        retval["ARM"]=camera[0]
-        retval["SPECTROGRAPH"]=int(camera[1])
-        retval["EXPID"]=expid
         retval["PANAME"]=paname
         retval["QATIME"]=datetime.datetime.now().isoformat()
+        retval["EXPID"] = "%08d"%frame.meta["EXPID"]
+        retval["CAMERA"] = frame.meta["CAMERA"]
+        retval["FLAVOR"] = frame.meta["FLAVOR"]
+        retval["NIGHT"] = frame.meta["NIGHT"]
         
         param = dict(
             PCHI_RESID=0.05, # P(Chi^2) limit for bad skyfiber model residuals
             PER_RESID=95.,   # Percentile for residual distribution
             )
-        qadict=qa_skysub(param,frame,skymodel)
-
-        #- Also need the med_resid_spec. Offline does not output this, so do again
-
-        skyfibers = np.where(frame.fibermap['OBJTYPE'] == 'SKY')[0]
-        residuals= frame.flux[skyfibers] - skymodel.flux[skyfibers] # Residuals
-        residuals_ivar = util.combine_ivar(frame.ivar[skyfibers], skymodel.ivar[skyfibers])
-
-        #- Now get the median residual for each fiber
-        med_resid_fiber=np.median(residuals,axis=1)
-        med_resid_wave=np.median(residuals,axis=0)
-        wavelength=frame.wave
+        qadict=qa_skysub(param,frame,skymodel,quick_look=True)
 
-        retval["VALUE"]={"MED_RESID":qadict["MED_RESID"], "NREJ": qadict["NREJ"], "NSKY_FIB": qadict["NSKY_FIB"], "RESID_PER": qadict["RESID_PER"], "NBAD_PCHI": qadict["NBAD_PCHI"], "MED_RESID_FIBER": med_resid_fiber, "MED_RESID_WAVE": med_resid_wave,"WAVELENGTH": wavelength}
+        retval["VALUE"]={"MED_RESID":qadict["MED_RESID"], "NREJ": qadict["NREJ"], "NSKY_FIB": qadict["NSKY_FIB"], "RESID_PER": qadict["RESID_PER"], "NBAD_PCHI": qadict["NBAD_PCHI"], "MED_RESID_FIBER": qadict["MED_RESID_FIBER"], "MED_RESID_WAVE": qadict["MED_RESID_WAVE"],"WAVELENGTH": qadict["WAVELENGTH"]}
```

Comment:
@profxj Agreed. I think we want whole dictionary here. I just pushed the change. Merging after the tests pass.